src/locale/locales/pt-BR/messages.po

```diff
@@ -630,7 +630,7 @@ msgstr "Altere o Seu Email"
 #: src/screens/Deactivated.tsx:73
 #: src/screens/Deactivated.tsx:77
 msgid "Check my status"
-msgstr ""
+msgstr "Verificar minha situação"
```

Comment:
```suggestion
msgstr "Verificar meu status"
```

**Status** is already used in the current translation, so I'd suggest using that instead of situação.

Reply:
```
#: src/view/screens/Settings.tsx:804
msgid "Status page"
msgstr "Página de status"
```

The other usage of Status is related to Bluesky's status page, in this case it's used in a queue context. The button will check if your account is already activated. "Verificar minha situação na fila" would be a better translation but "fila" is already used in the screen, so I didn't want to repeat it there